extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -1794,6 +1794,9 @@ public Long apply(Integer key, Long existingOffsetInMap)
               ServiceMetricEvent.builder().setDimension("dataSource", dataSource).build("ingest/kafka/lag", lag)
           );
         }
+        catch (InterruptedException e) {
+          log.warn("Interrupted while computing Kafka lag");
```

Comment:
Just doing nothing would be fine, I think. I can't see a need for a log message. Up to you though @pjain1, I'm fine with it either way.